code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -136,8 +140,43 @@ namespace <#= qualifiedNamespace #>
                     entityManager.AddComponentData(entity, updates);
                 }
 
-                // TODO: Events?
+<# if (eventDetailsList.Count > 0) { #>
+                // Events
```

Comment:
Unnecessary comment imo

Reply:
I quite like having something which says what different blocks are for in codegen. Keeps different features easier to isolate. Ultimately it would be nice to have events and components codegen only as coupled as they have to be. But not something to do now.